cmd/aws-s3-provisioner.go

```diff
@@ -33,8 +33,8 @@ import (
 	// "github.com/aws/aws-sdk-go-v2/service/iam"
 
 	"github.com/golang/glog"
-	"github.com/yard-turkey/lib-bucket-provisioner/pkg/apis/objectbucket.io/v1alpha1"
-	libapi "github.com/yard-turkey/lib-bucket-provisioner/pkg/api"
+	bktV1 "github.com/yard-turkey/lib-bucket-provisioner/pkg/apis/objectbucket.io/v1alpha1"
+	types "github.com/yard-turkey/lib-bucket-provisioner/pkg/api"
```

Comment:
`types` is pretty vague.  In the k8s world, it's usually reserved for k8s API types.  `libapi` was more inline with the contents of the import.   Something more concise would be even better.

Reply:
agree